src/Core/Content/Category/Event/SalesChannelEntryPointsEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Category\Event;
+
+use Shopware\Core\System\SalesChannel\SalesChannelEntity;
+
+class SalesChannelEntryPointsEvent
+{
+    /**
+     * @var array
+     */
+    protected $ids;
+
+    /**
+     * @var SalesChannelEntity
+     */
+    protected $salesChannelEntity;
+
+    public function __construct(array $ids, SalesChannelEntity $salesChannelEntity)
+    {
+        $this->ids = $ids;
```

Comment:
what does this array contain? I think it would be useful do add some PHPStan/Psalm like Annotations or at least some PHPdoc comment to explain. Also `ids` is not quite descriptive :blush: